h/static/scripts/directive/annotation.coffee

```diff
@@ -61,19 +61,27 @@ AnnotationController = [
 
     model = $scope.annotationGet()
 
+    # Set the permissions of a new anonotation.
+    setPermissions = ->
```

Comment:
I think the naming could be a little clearer - `setDefaultPermissions`. In functions in this class you also need to be explicit about whether the changes apply to the _domain_ model (`model`) or the _view_ model (`@annotation`). In the context I think what we want to do is apply the changes to `model` and then let `render()` update the view.